src/utils/review.ts

```diff
@@ -0,0 +1,29 @@
+import { AdminReview } from "@/services/api/admin/useReviews";
+
+function isGreaterThan24Hours(date: Date): boolean | null {
+  if (!date) return null;
+  // Get the current date and time
+  const createdReview = new Date(date);
+  const now: Date = new Date();
```

Comment:
No need for explicit type assertion since typescript can automatically infer the type
```suggestion
  const now = new Date();
```